share/spice/code_day/code_day.js

```diff
@@ -0,0 +1,63 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_code_day = function(api_result){
+        if (!api_result || api_result.length === 0) {
+            return;
+        }
+
+        DDG.require(['maps', 'moment.js'], function(){
+            Spice.add({
+                id: "code_day",
+                name: "Events",
+                data: api_result,
+                model: "Place",
+                view: "Places",
+                normalize: function(event){
+                    if(!event.current_event) return null;
+                    var venue = event.current_event.venue,
+                        startDate = moment(event.current_event.starts_at * 1000), // convert from unix epoch
+                        endDate = moment(event.current_event.ends_at * 1000);
+                    
+                    return {
+                        id: event.id,
+                        data_front: {
+                            showPin: true,
+                            title: "CodeDay " + event.name,
+                            altSubtitle: startDate.format("MMMM Do") + " - " + endDate.format("Do") + ", noon - noon",
+                            titleClass: 'tile__title--3 tx--16 tx--bold mg--none',
+                            altSubClass: 'tx--13 tx-clr--grey'
+                        },
+                        data_back: {
+                            title: "CodeDay " + event.name,
+                            url: event.current_event.urls.home,
+                            description: venue.address.city + ", " + venue.address.state,
```

Comment:
> We should probably also indicate the # of tickets remaining

How about we only show it when it's less than 20? I was considering this earlier but someone might search it, see "87 Tickets Remaining" and think "oh I have time I'll register later" and then forget about it.